io.github.baldeuniversel.am-okay.yml

```diff
@@ -0,0 +1,44 @@
+id: io.github.baldeuniversel.am-okay
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk 
+
+command: am-okay
+
+finish-args:
+    - --filesystem=host
+
+modules:
+    - name: am-okay-package
+      buildsystem: simple      
+      sources:
+          - type: git 
+            url: https://github.com/baldeuniversel/am-okay.git 
+            branch: new-pr
```

Comment:
yet another time: you can not have a git branch here. This is not an optional suggestion.